packages/runner/injection/patches/cookies.ts

```diff
@@ -63,6 +63,17 @@ export const patchDocumentCookie = (requestCookies: AutomationCookie[]) => {
     })
   }
 
+  const setCookie = (cookie: ToughCookie | string) => {
```

Comment:
This is a (somewhat) incidental fix that I discovered while working on `cy.setCookie()`. The `document.cookie` patch should not throw an error when attempting to set a cookie with a different domain, which matches the behavior of the real `document.cookie`.